platform/engineering/request-for-comment/2022-02-23-remove-gender-fax-from-mobile-api

```diff
@@ -0,0 +1,23 @@
+RFC: Removing gender and fax number from mobile api
+Date: 2022-02-23
+Related Issue: department-of-veterans-affairs/va-mobile-app#2565
+Background
+Currently gender and fax are both returned from the mobile API and displayed in the app. 
+
+Motivation
+Profile is removing gender and fax number. More details: https://dsva.slack.com/archives/C018V2JCWRJ/p1643395405843779
+
+Design
+The proposed change for the API would be to add an accessor to the iam_user model to pull gender from the IAM identity or return null if it doesn't exist. This would remove the risk of the profile changes breaking the mobile API by making gender no longer accessible through the user model. Otherwise we propose doing nothing in the API until we are certain these fields have been completely removed from the upstream services and confident most users have updated the app beyond the version where the app no longer shows the fields.
+
```

Comment:
I also think that – probably outside of this RFC – we should have an ultimate goal of providing these services inside vets-api through another layer of abstraction and versioning.